osclib/adi_command.py

```diff
@@ -87,9 +87,13 @@ def create_new_adi(self, wanted_requests, by_dp=False, split=False):
             print(Fore.YELLOW + (group if group != '' else 'wanted') + Fore.RESET)
 
             name = None
+            nonfree_repo_required = False
```

Comment:
Why have a loop variable when it is simply:

```python
nonfree = target_project == self.api.cnonfree
```

As for variable name, same suggestion as on `create_adi_project()`.